include/meta/meta.hpp

```diff
@@ -2139,8 +2139,9 @@ namespace meta
             struct find_if_<list<List...>, Fun,
                             void_<integer_sequence<bool, bool(invoke<Fun, List>::type::value)...>>>
             {
-#if defined(__clang__) && __clang_major__ < 6
+#if (defined(__clang__) && __clang_major__ < 6) || defined(__apple_build_version__)
```

Comment:
There are roughly half a dozen occurrences of `// Explicitly specify extent to avoid https://llvm.org/bugs/show_bug.cgi?id=28385` in here which need to be fixed similarly.

Reply:
Right, I haven't looked for more occurrences. My code base used `meta::find_if` directly at several places. I've searched this file for the string '28385' and fixed it at every place which I've found that way.

    grep -Rn '28385' .
    ./meta/meta.hpp:1994:                // Explicitly specify extent to avoid https://llvm.org/bugs/show_bug.cgi?id=28385
    ./meta/meta.hpp:2052:                // Explicitly specify extent to avoid https://llvm.org/bugs/show_bug.cgi?id=28385
    ./meta/meta.hpp:2145:                // Explicitly specify extent to avoid https://llvm.org/bugs/show_bug.cgi?id=28385
    ./meta/meta.hpp:2206:                // Explicitly specify extent to avoid https://llvm.org/bugs/show_bug.cgi?id=28385
    ./meta/meta.hpp:2334:                // Explicitly specify extent to avoid https://llvm.org/bugs/show_bug.cgi?id=28385
    ./meta/meta.hpp:2379:                // Explicitly specify extent to avoid https://llvm.org/bugs/show_bug.cgi?id=28385